tests/test_elegant_conversion.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import torch
+
+import cheetah
+from cheetah.utils import is_mps_available_and_functional
+
+
+def test_fodo():
+    file_path = "tests/resources/fodo.lte"
+    converted = cheetah.Segment.from_elegant(file_path, "fodo")
+
+    correct_lattice = cheetah.Segment(
+        [
+            cheetah.Quadrupole(
+                name="q1", length=torch.tensor(0.1), k1=torch.tensor(1.5)
+            ),
+            cheetah.Drift(name="d1", length=torch.tensor(1)),
+            cheetah.Marker(name="m1"),
+            cheetah.Dipole(name="s1", length=torch.tensor(0.3), e1=torch.tensor(0.25)),
+            cheetah.Drift(name="d1", length=torch.tensor(1)),
+            cheetah.Quadrupole(
+                name="q2", length=torch.tensor(0.2), k1=torch.tensor(-3)
+            ),
+            cheetah.Drift(name="d2", length=torch.tensor(2)),
+        ],
+        name="fodo",
+    )
+
+    assert converted.name == correct_lattice.name
+    assert [element.name for element in converted.elements] == [
+        element.name for element in correct_lattice.elements
+    ]
+    assert converted.q1.length == correct_lattice.q1.length
+    assert converted.q1.k1 == correct_lattice.q1.k1
+    assert converted.q2.length == correct_lattice.q2.length
```

Comment:
This test is missing a docstring. 

I'm also a little surprised that it works despite properties only being defined as scalars. I think they should be defined as single-element vectors for now (even though #208 should hopefully and eventually remove the need for that).

Reply:
I think this works since the elements properties are only read when calling `segment.track()`, and constructing scalar valued elements is perfectly fine. But of course you are right that the test should be done properly.